### PR TITLE
post about Minimal Viable Governance

### PR DESCRIPTION
**Pull request by johannbarbie**
delivers https://github.com/leapdao/leapdao-website/issues/15

### CODE REVIEW

File: src/blog/content/Minimal-Viable-Governance.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+**Parent Block Interval -** The minimum distance of Plasma blocks in terms of Ethereum blocks.
+**Register Token -** The Plasma chain can color its transactions in any ERC20 token that exists on the main-net. The token must be registered before in can be deposited to the chain.
+
+**Upgrade Plasma Bridge -** This function switches out the implementation logic of the Bridge itself. This way new features or bug-fixes can be introduced.
+
+**Change Governance Process -** This function allows to switch to another governance process all together. This could be governance by token vote or holographic consensus.
+
+## Limitation and Outlook
+
+The LeapDAO community strives to launch a network with the governance process described above on the Ethereum main-net in January 2019. Once the network is established updates will be processed through Minimal Viable Governance to add features to the network.
```
[the remark this answers is not in the document]
Comment (johannbarbie) [reply src/blog/content/Minimal-Viable-Governance.md]: the use of LeapDAO is kind of strange here. i think you mean the guys that work for the project, while what it should be is the totality of all stakeholders.

File: src/blog/content/Minimal-Viable-Governance.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Like any minimal viable product, so is our Minimal Viable Governance implementation limited to an essential feature, giving users a choice to accept a change or exit before. It is lacking in many other ways, for example:
+
+- The proposal page is currently hosted, and hence needs to be trusted by the users. Instructions how to operate your own node and connect it to a local version of the proposals page will be published soon.
```
[the remark this answers is not in the document]
Comment (johannbarbie) [reply src/blog/content/Minimal-Viable-Governance.md]: the use of LeapDAO is kind of strange here. i think you mean the guys that work for the project, while what it should be is the totality of all stakeholders.

**Comment by johannbarbie** [comment]
@troggy applied all your suggestions, except the `leapdao` stuff.

File: src/blog/content/Minimal-Viable-Governance.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (troggy) [reply src/blog/content/Minimal-Viable-Governance.md]: Yes, that's what I meant. Just wanted to outline that no change can skip the governance contract by design

[the remark this answers is not in the document]
Comment (troggy) [reply src/blog/content/Minimal-Viable-Governance.md]: I see that you addressed it already 👍

**Pull request merged**